projects/igniteui-angular/src/lib/directives/text-selection/text-selection.directive.ts

```diff
@@ -108,7 +108,7 @@ export class IgxTextSelectionDirective {
 
     public trigger() {
         if (this.selected && this.nativeElement.value.length) {
-                this.nativeElement.select();
+            requestAnimationFrame(() => this.nativeElement.select());
```

Comment:
Would actually like a comment above explaining why the rAF is used. Otherwise, I believe this will do